assets/index.scss

```diff
@@ -844,3 +861,14 @@ span.bold {
   @extend .lc-color-like-green;
   @extend .lc-font-weight-600;
 }
+
+
+.simple-svg-wrapper {
+  display: flex;
```

Comment:
seems this broke MissionDIalog's icon

Reply:
nvm missed https://github.com/likecoin/likecoin-store/pull/427/commits/0397b5f72c786553b96a0dc610e30f13187b1560